openfecwebapp/config.py

```diff
@@ -28,7 +28,7 @@
 )
 
 features = {
-    'legal': bool(env.get_credential('FEC_FEATURE_LEGAL', '')),
+    'legal_statutes': bool(env.get_credential('FEC_FEATURE_LEGAL_STATUTES', '')),
```

Comment:
I agree that we shouldn't upgrade the README everytime we change the feature flags but the README already links to `config.py` https://github.com/18F/openFEC-web-app#features